### PR TITLE
fix: Manual execution and vault dashboard values

### PR DESCRIPTION
**Pull request by savudani8**

### CODE REVIEW

File: src/pages/vault-dashboard/vault-dashboard.page.tsx [hunk 1]
```diff
@@ -157,12 +158,12 @@ function VaultDashboard(): JSX.Element {
   const VAULT_ITEMS = [
     {
       title: t('vault.locked_collateral'),
-      value: collateral,
+      value: safeRoundTwoDecimals(collateral),
```
Comment (nud3l) [inline src/pages/vault-dashboard/vault-dashboard.page.tsx:161]: we have a `displayBtcAmount` and a `displayDotAmount` function -> I'd prefer those here so we have one place to change currencies.

**Review by nud3l — APPROVED** [review]

File: src/pages/vault-dashboard/vault-dashboard.page.tsx [hunk 2]
```diff
@@ -13,6 +13,7 @@ import {
 } from '@interlay/polkabtc';
 import { useTranslation } from 'react-i18next';
 import tw from 'twin.macro';
+import { IssueColumns, RedeemColumns } from '@interlay/polkabtc-stats';
```
Comment (anton-karlovskiy) [inline src/pages/vault-dashboard/vault-dashboard.page.tsx:16]: @savudani8 
Could do line breaking (for more than 2).

File: src/pages/vault-dashboard/vault-dashboard.page.tsx [hunk 3]
```diff
@@ -40,7 +40,7 @@ import './vault-dashboard.page.scss';
 import VaultIssueRequestsTable from 'containers/VaultIssueRequestTable';
 import VaultRedeemRequestsTable from 'containers/VaultRedeemRequestTable';
 import usePolkabtcStats from 'common/hooks/use-polkabtc-stats';
-import { IssueColumns, RedeemColumns } from '@interlay/polkabtc-stats';
+import { displayBtcAmount, displayDotAmount, safeRoundTwoDecimals } from 'common/utils/utils';
```
Comment (anton-karlovskiy) [inline src/pages/vault-dashboard/vault-dashboard.page.tsx:43]: @savudani8
Could do line breaking (for more than 2).

**Review by anton-karlovskiy — COMMENTED** [review]
Overall looks good to me.
Two minor comments. @savudani8

File: src/pages/vault-dashboard/vault-dashboard.page.tsx (hunk 2 above)
Comment (savudani8) [reply src/pages/vault-dashboard/vault-dashboard.page.tsx:16]: Thanks for reviewing! I'll keep the suggestion in mind next time

**Pull request merged**